### ISSUE TITLE
Can't install 2.0.2

### ISSUE THREAD

**Issue by nkovacs** [BenoitZugmeyer/eslint-plugin-html#54]
yarn add eslint-plugin-html:

```
error An unexpected error occurred: "https://registry.yarnpkg.com/eslint-plugin-html/-/eslint-plugin-html-2.0.2.tgz: Invalid tar header. Maybe the tar is corrupted or it needs to be gunzipped?".
```

npm install eslint-plugin-html:

```
npm ERR! cb() never called!
```

yarn version is 0.23.3. Npm version is 4.5.0

**Comment by nkovacs** [comment]
Duplicate of #53
